code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -457,6 +457,12 @@
 		if(istype(attacker))
 			src.harmedBy(attacker)
 
+/mob/living/critter/flock/drone/hitby(atom/movable/AM, datum/thrown_thing/thr)
+	. = ..()
+	var/mob/attacker = thr.user
+	if(istype(attacker) && !istype(attacker, /mob/living/critter/flock/drone))
```

Comment:
```suggestion
	if(istype(attacker) && !isflock(attacker))
```

Reply:
I only did it that way because I don't think Flockbits can pick up items, but idk

Reply:
Use isflock